[09:05:10 AM] [INFO] logs/06-03-24_09-05-10.log
[09:05:14 AM] [ERROR] unable to load game: D:/Downloads/App/Wheres My Water Hack New Clean 1.2 [Level S6-6, 5 Spout with OffsetVariation and ExpulsionAngleVariation (Change III)] + Change original object textures
Traceback (most recent call last):
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 1658, in loadGame
    self.game = wmwpy.load(
                ^^^^^^^^^^^
  File "D:\Downloads\Data\Python\Lib\site-packages\wmwpy\__init__.py", line 56, in load
    return GAMES.get(game, Game)(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Downloads\Data\Python\Lib\site-packages\wmwpy\gametemplate.py", line 104, in __init__
    super().__init__(gamepath = gamepath, assets = assets, db = db, profile = profile, baseassets = baseassets, load_callback = load_callback)
  File "D:\Downloads\Data\Python\Lib\site-packages\wmwpy\game.py", line 139, in __init__
    this.updateFilesystem(load_callback = load_callback)
  File "D:\Downloads\Data\Python\Lib\site-packages\wmwpy\game.py", line 148, in updateFilesystem
    this.filesystem.getAssets(load_callback = load_callback)
  File "D:\Downloads\Data\Python\Lib\site-packages\wmwpy\utils\filesystem.py", line 144, in getAssets
    load_callback(current, path, total)
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 514, in updateProgressBar
    self.progress_bar['progress_bar']['max'] = max
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "D:\Downloads\Data\Python\Lib\tkinter\__init__.py", line 1732, in __setitem__
    self.configure({key: value})
  File "D:\Downloads\Data\Python\Lib\tkinter\__init__.py", line 1721, in configure
    return self._configure('configure', cnf, kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Downloads\Data\Python\Lib\site-packages\tkmacosx\utils\colorvar_patches.py", line 106, in _patch
    return fn(self, cmd, cnf, kw)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Downloads\Data\Python\Lib\tkinter\__init__.py", line 1711, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: invalid command name ".!frame.!progressbar"
[09:05:14 AM] [ERROR] WME ended prematurely
Traceback (most recent call last):
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 1787, in main
    app = WME(None)
          ^^^^^^^^^
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 207, in __init__
    self.loadGame()
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 1667, in loadGame
    self.state = 'enabled'
    ^^^^^^^^^^
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 535, in state
    self.enableWindow()
  File "D:\Downloads\My app\wheres-my-editor\src\main.py", line 352, in enableWindow
    self.level_canvas.bind("<MouseWheel>", self.onLevelMouseWheel)
  File "D:\Downloads\Data\Python\Lib\tkinter\__init__.py", line 1450, in bind
    return self._bind(('bind', self._w), sequence, func, add)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Downloads\Data\Python\Lib\tkinter\__init__.py", line 1404, in _bind
    self.tk.call(what + (sequence, cmd))
_tkinter.TclError: can't invoke "bind" command: application has been destroyed
